core/src/avm1/globals/bitmap_data.rs

```diff
@@ -193,7 +193,8 @@ fn get_pixel<'gc>(
             if let (Some(x_val), Some(y_val)) = (args.get(0), args.get(1)) {
                 let x = x_val.coerce_to_u32(activation)?;
                 let y = y_val.coerce_to_u32(activation)?;
-                let col = operations::get_pixel(bitmap_data, x, y);
+                // AVM1 returns a signed int, so we need to convert it.
```

Comment:
can we have some test for the negative values here?

Reply:
There are a lot of existing avm1 tests that fail without this.